spec/lib/oauth/error_spec.rb

```diff
@@ -17,5 +18,21 @@
       )
       error.description
     end
+
+    context "when there are variables" do
+      subject(:error) do
+        described_class.new(
+          :invalid_code_challenge_method,
+          :some_state,
+          {
+            challenge_methods: "foo, bar"
+          }
+        )
+      end
+
+      it "is translated from translation messages with variables" do
+        expect(error.description).to eq("The code challenge method must be one of foo, bar.")
+      end
+    end
```

Comment:
We should have a test case for the "one" option, i.e., just "foo" not "foo, bar"

(this likely more applies to pre_authorization_spec.

Reply:
I added a test for `%w(foo)` and `[]` in ccc97732f3977e57d3a7c541689c30f418de6697